src/component/Marketplace/index.jsx

```diff
@@ -7,6 +7,48 @@ import CollectionCard from '../Home/Cards/CollectionCard';
 import DiscoverCard from '../Home/Cards/DiscoverCard';
 
 function Marketplace() {
+  const inputRef = useRef();
```

Comment:
Why is your Searchbar code added here in the Stepper branch? Didnt you branch it from the Main branch?